### PR TITLE
darkmode/semesterly-logo

### PR DESCRIPTION
**Pull request by K02D**
Before:
image

After:
image

### CODE REVIEW

File: static/js/redux/ui/TopBar.tsx [hunk 1]
```diff
@@ -25,6 +25,7 @@ const TopBar = () => {
   const userInfo = useAppSelector((state) => state.userInfo.data);
   const currentSemester = useAppSelector((state) => getCurrentSemester(state));
   const isComparing = useAppSelector((state) => state.compareTimetable.isComparing);
+  const theme = useAppSelector((state) => state.theme.theme.name);
```
Comment (JJamesWWang) [inline static/js/redux/ui/TopBar.tsx]: You can use the selectTheme selector that Jacky wrote here

Comment (K02D) [reply static/js/redux/ui/TopBar.tsx]: Changed

**Review by JJamesWWang — APPROVED** [review]
看起来蛮好的

**Review by JJamesWWang — APPROVED** [review]
看起来蛮好的

**Pull request merged**